assets/training/vision/environments/automl-dnn-vision-gpu/environment.yaml

```diff
@@ -0,0 +1,11 @@
+image:
+  name: azureml/curated/automl-dnn-vision-gpu
+  os: linux
+  context:
+    dir: context
+    dockerfile: Dockerfile
+    template_files:
+    - Dockerfile
+  publish:
+    location: mcr
+    visibility: unlisted
```

Comment:
The mcr image is added in curated-unlisted.yaml file here: https://github.com/microsoft/mcr/blob/main/teams/azureml/base/curated-unlisted.yml
Could you please clarify if this would have an impact if the visibility is changed to public.
Also, in the future AutoML team is planning to switch from CEs hosted by EMS team to CEs in registry and would use this image. Do you think that would cause an issue, if visibility is changed to public.
fyi: @vivijay91

Reply:
The image visibility would need to be changed to public in the mcr repo as well and moved to curated-public.yml. This is a requirement for us for environments in system registries largely because it's very difficult to troubleshoot MCR images that are unlisted, and with a public environment referencing the image there's really no need for the image to be unlisted. Any additional thoughts from you, @vizhur?